frontend/automation/pages/pages.ts

```diff
@@ -0,0 +1,23 @@
+import type { Page } from '@playwright/test'
```

Comment:
This class is so we do not have to instantiate in every test file. Let me know if there is a better idea or if you'd rather have the instantiation left as is. Easy either way.